ppa/archive/import_util.py

```diff
@@ -247,7 +255,15 @@ def pairtree_paths(self):
     def rsync_data(self):
         """Use rsync to retrieve data for the volumes to be imported."""
 
-        logger.info("rsyncing pairtree data for %s", ", ".join(self.source_ids))
+        # limit the number of ids included in the log message
+        log_detail = ""
+        rsync_count = len(self.source_ids)
+        if rsync_count <= 12:
+            log_detail = ", ".join(self.source_ids)
+        else:
+            log_detail = "%d volumes" % rsync_count
+
+        logger.info("rsyncing pairtree data for %s", log_detail)
```

Comment:
previously this was logging _all_ of the source ids, which worked for small-batches of imports or rsync but is obviously unhelpful when it's all the hathitrust records in ppa

Reply:
What's the reasoning for setting the default to 12 specifically?

Reply:
@laurejt totally arbitrary! Just a guess at what might be reasonable. I don't think we want more than a few lines of logging worth of ids, it's not particularly meaningful and it's too noisy. (I did not actually check how this looks in the log files). Could change to  10 or 15 if you think that would be a less distracting number. 😁